templates/admin/user/edit.tmpl

```diff
@@ -74,7 +74,7 @@
 				<div class="inline field">
 					<div class="ui checkbox">
 						<label><strong>{{.i18n.Tr "admin.users.prohibit_login"}}</strong></label>
-						<input name="prohibit_login" type="checkbox" {{if .User.ProhibitLogin}}checked{{end}}>
+						<input name="prohibit_login" type="checkbox" {{if .User.ProhibitLogin}}checked{{end}} {{if or (eq .User.ID .SignedUserID) (eq .User.ID 1)}}disabled{{end}}>
```

Comment:
Probably best to check if user is an admin instead of hard coding UID 1? Because I'm the only user on my Gitea install, but I had to delete my first user so user ID of 1 doesn't exist.

Reply:
yes, I see. That mans all admin can't bee limited to login. But I think maybe we should have only one main admin who can add or remove admin or even limit admin to login.

Reply:
and the main admin id should be seting by app.ini.

Reply:
@a1012112796 I don't think we need that. The last admin can not update himself.